### PR TITLE
First step towards adding threat privacy group support.

### PR DESCRIPTION
**Pull request by mgoffin**
:ok: to test!

This adds the ThreatPrivacyGroup class for management of Threat Privacy
Groups. Through inheritance it allows you to add new Threat Privacy
Groups as well as update them. There are helper methods for finding out
which Groups you are a member of, owner of, getting members of a Threat
Privacy Group, as well as set the members of a Threat Privacy Group.

Also added support for the `include_expired`, `max_confidence`, and `min_confidence`
searching parameters. Adjusted the `owner` docstrings to note that you can
comma-delimit the value to search for multiple owners.

Going to bump to version 0.3.1 for this. If there's a few small other changes
coming soon we can wait to release 0.3.1 until things are ready :)

### CODE REVIEW

File: pytx/pytx/__init__.py [hunk 1]
```diff
@@ -17,5 +18,6 @@
     'MalwareFamily',
     'ThreatExchangeMember',
     'ThreatDescriptor',
-    'ThreatIndicator'
+    'ThreatIndicator',
+    'ThreatPrivacyGroup'
```
Comment (jessek) [inline pytx/pytx/__init__.py]: Add a trailing comma to avoid changing the blame log later on.

File: pytx/pytx/common.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def __setattr__(self, name, value):
 
         object.__setattr__(self, name, value)
         if name == c.ID:
-            self._DETAILS = t.URL + value + '/'
+            self._DETAILS = t.URL + t.VERSION + value + '/'
```
Comment (jessek) [inline pytx/pytx/common.py]: Er... won't this result in something like:

    graph.facebook.com/2.499999999999/

Shouldn't there be a slash between the version and the value? Or am I reading this wrong?

File: pytx/pytx/threat_privacy_group.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+from common import Common
+from request import Broker
+from vocabulary import ThreatPrivacyGroup as tpg
+from vocabulary import ThreatExchange as t
+from errors import pytxValueError
+
+
+class ThreatPrivacyGroup(Common):
+
+    _URL = t.URL + t.VERSION + t.THREAT_PRIVACY_GROUPS
```
Comment (jessek) [inline pytx/pytx/threat_privacy_group.py]: Again, same issue with the lack of slashes?

File: pytx/pytx/vocabulary.py [hunk 4]
```diff
@@ -15,9 +15,15 @@ class ThreatExchange(object):
     THREAT_EXCHANGE_MEMBERS = 'threat_exchange_members/'
     THREAT_DESCRIPTORS = 'threat_descriptors/'
     THREAT_INDICATORS = 'threat_indicators/'
+    THREAT_PRIVACY_GROUPS = 'threat_privacy_groups/'
+    THREAT_PRIVACY_GROUPS_MEMBER = 'threat_privacy_groups_member/'
+    THREAT_PRIVACY_GROUPS_OWNER = 'threat_privacy_groups_owner/'
 
+    INCLUDE_EXPIRED = 'include_expired'
```
Comment (jessek) [inline pytx/pytx/vocabulary.py]: Let's keep these in alphabetical order

File: pytx/pytx/common.py (hunk 2 above)
Comment (mgoffin) [inline pytx/pytx/common.py]: The VERSION vocabulary attribute is 'v2.5/'

Comment (jessek) [inline pytx/pytx/common.py]: [[sheepish]]

**Comment by mgoffin** [comment]
This should be ready :)

**Pull request merged**